README.md

```diff
@@ -34,6 +34,8 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+You will also need to add your own TRELLO_API_KEY and TRELLO_API_TOKEN to the .env file as this will not be stored between sessions.
```

Comment:
You could put these two variables into the .env.template file with placeholder values to make it slightly simpler for the reader to get set up.

And perhaps common sense, but the reader needs a Trello account first.

Reply:
Have added this to .env.template file